src/reconcile/caliperEventTsv.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh --
+
+# Send input through stdin or give a filename as an argument
+
+jq -r '[.eventTime[:19]+"Z", .actor.name, .object["@id"], .generated.value//"0"]|@tsv' ${1:--} | \
+    sort | sed 's/%40/@/g'
```

Comment:
The PR app. log (stored in its MySQL DB) contains only information analogous to Caliper events with the AssessmentItemEvent type and the Completed or Skipped actions.  According to @pushyamig, the log doesn't contain information about other Caliper event types that the application is emitting, like AnnotationEvent, AssessmentEvent, NavigationEvent, or SessionEvent.

Reducing the dataset using grep is one of the advantages of working with a JSONL file.  Rather than a file containing valid JSON end-to-end, we have a file with one object per line.  It helps to select only a few objects (millions) out of many (millions more).